(SCHEDULE) (OLD) check if ansible version is under 2.12.0]

When i run my playbook with default Settings, i got the following error:

Error message: 
TASK [l3d.restic : (SCHEDULE) (OLD) check if ansible version is under 2.12.0] *********************************************************************************************************************************
fatal: [192.168.1.193 -> localhost]: FAILED! => {
    "assertion": "ansible_version.full is version_compare('2.12.0', '<')",
    "changed": false,
    "evaluated_to": false,
    "msg": "[ERROR] Youre ansible version is above 2.12.0"
}



ansible --version
ansible [core 2.14.6]
  config file = None
  configured module search path = ['/home/kroon/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  ansible collection location = /home/kroon/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.10.6 (main, May 29 2023, 11:10:38) [GCC 11.3.0] (/usr/bin/python3)
  jinja version = 3.0.3
  libyaml = True

--- a/defaults/main.yml
+++ b/defaults/main.yml
@@ -21,7 +21,7 @@ restic_systemd_timer_default_oncalendar: '*-*-* 02:00:00'
 
 # perform simple version check for this role? (true is recomended)
 submodules_versioncheck: false
-restic_do_not_cleanup_cron: false
+restic_do_not_cleanup_cron: true
 
 # cache
 restic__cache_config: false

